### DISCUSSION TITLE
Request for Custom CircuitPython Version for EDU PICO Board Compatibility

### DISCUSSION THREAD

**Discussion by SalamCytron** [thonny/thonny#3181] [General]
Hi, 

I have a request on behalf of Cytron Technologies regarding our [EDU PICO educational product](https://www.cytron.io/p-edu-project-and-innovation-kits-for-pico-w), which uses Thonny IDE and CircuitPython in our lessons.

We're grateful for Thonny IDE's beginner-friendly interface – it's been a huge help.

However, we've encountered compatibility issues with the latest CircuitPython versions and our example code lessons due to firmware changes. After testing, we found a stable version that works perfectly:

```Adafruit CircuitPython 9.0.0-beta.2 on 2024-02-20; Cytron EDU PICO W with rp2040```

Could you please include this version in the installation option? It would make a big difference for us and our users, ensuring a smooth experience, especially for beginners.

Thank you for considering our request.

Best regards,

Salam
Cytron Technologies

**Comment by aivarannamaa** [comment]
I now added the older version to the download list of this device. Please let me know when this is not required anymore!

**Reply by SalamCytron** [reply]
Thank you so much. Its really helped a lot.
